src/components/chats/ChatList/ChatList.tsx

```diff
@@ -89,6 +90,19 @@ function ChatListContent({
     loadMore
   )
 
+  useEffect(() => {
+    ;(async () => {
+      const commentIdFromUrl = getUrlQuery('chatId')
+      if (!commentIdFromUrl) return
+
+      const element = await getRepliedElement(commentIdFromUrl)
```

Comment:
Rename the action to more common, like getHistoricalElement or something like this